app/scripts/controllers/HomeCtrl.js

```diff
@@ -2,6 +2,14 @@
   function HomeCtrl(Room, $uibModal) {
 
     this.roomObject = Room;
```

Comment:
I do need this line in some sort of from because in the ```home.html``` I am using ```home.roomObject.all```; however, I can create an ```all``` property in my home controller and set it to ```Room.all```.